src/functions/edi/inbound/handler.ts

```diff
@@ -6,7 +6,7 @@ import {
   DeleteObjectCommand,
   GetObjectCommand,
 } from "@stedi/sdk-client-buckets";
-import * as x12 from "@stedi/x12-tools/node.js";
+import * as x12 from "@stedi/x12-tools";
```

Comment:
not directly related to this PR, but picked up the latest `x12-tools` which no longer requires pointing to the `node` module